src/com/epam/homework/lists/StudentList.java

```diff
@@ -0,0 +1,84 @@
+package com.epam.homework.lists;
+
+public class StudentList {
+    private static final int DEFAULT_CAPACITY = 10;
+
+    private int size = 0;
+
+    private Student[] students;
+
+    public StudentList() {
+        students = new Student[DEFAULT_CAPACITY];
+    }
+
+    public StudentList(int capacity) {
+        if (capacity < 0) {
+            throw new IllegalArgumentException("Illegal capacity: " + capacity);
+
+        }
+        students = new Student[capacity];
+    }
+
+    public void add(Student student) {
```

Comment:
I tried to add 20 Students to the `StudentList` and got Exception while removing.

Reply:
Forgot to add break  in StudentList remove() method to terminate loop after deleting first Student that matches deletion criteria. Loop was iterating through all list deleting all Students that match that criteria and the list size decremented. Then when trying to get object via index to delete, it was throwing exception. Now it is fixed.